src/content/protocols/kodiak-v3.md

```diff
@@ -0,0 +1,230 @@
+| KodiakFarm          | setStakingPaused           | Pauses new staking. Malicious owner could permanently pause staking, blocking new deposits.                                          | Safe Wallet 1 |
+| KodiakFarm          | setRewardsCollectionPaused | Pauses reward collection. Malicious owner could block all reward claims.                                                             | Safe Wallet 1 |
+| KodiakFarm          | addNewRewardToken          | Adds new reward tokens. Malicious owner could add malicious tokens or set extreme rates.                                             | Safe Wallet 1 |
+| KodiakFarm          | setStakingTokenCap         | Sets maximum staking amount. Malicious owner could set extremely low caps, limiting farm capacity.                                   | Safe Wallet 1 |
+| PandaFactory        | renounceOwnership          | Removes admin control. Prevents future parameter changes. Could permanently lock token launch functionality.                         | Safe Wallet 3 |
+| PandaFactory        | transferOwnership          | Transfers admin control. New owner gains control over all token launch parameters.                                                   | Safe Wallet 3 |
+| PandaFactory        | setMinRaise                | Sets minimum fundraising threshold. Malicious owner could set unreasonably high minimums, preventing new launches.                   | Safe Wallet 3 |
+| PandaFactory        | setMinTradeSize            | Sets minimum trade size. Malicious owner could set extreme minimums, making pools unusable.                                          | Safe Wallet 3 |
+| PandaFactory        | setTreasury                | Changes fee recipient. Malicious owner could redirect all protocol fees.                                                             | Safe Wallet 3 |
+| PandaFactory        | setDexFactory              | Sets DEX factory for graduated tokens. Malicious owner could point to malicious DEX, stealing funds during graduation.               | Safe Wallet 3 |
+| PandaFactory        | setAllowedImplementation   | Controls pool implementations. Malicious owner could whitelist malicious pool contracts.                                             | Safe Wallet 3 |
+| PandaFactory        | setWbera                   | Sets wrapped BERA address. Malicious owner could set malicious WBERA, breaking native swaps.                                         | Safe Wallet 3 |
+| PandaFactory        | setIncentive               | Sets launch incentives. Malicious owner could set extreme incentives, draining protocol reserves.                                    | Safe Wallet 3 |
+| PandaFactory        | setPandaPoolFees           | Sets pool fee structure. Malicious owner could set extreme fees, making pools unviable.                                              | Safe Wallet 3 |
```

Comment:
fees cannot lead to loss of funds for panda token (immutable after deploying) please add this